src/popup/views/app.vue

```diff
@@ -127,6 +134,22 @@ export default {
         data: script,
       }));
     },
+    failureReason() {
```

Comment:
What's this for? I didn't find a place using it.

Reply:
At the start of the template - `:data-failure-reason="failureReason">` which enables custom styling of the entire app depending on the state.

Reply:
Do you mean that users can apply different styles depending on the state? I thought you were writing CSS for it and didn't find any. 😅

Reply:
Yep, it's for customCSS. We did it for all extension pages in https://github.com/openstyles/stylus